### PR TITLE
Open external links in the default browser 

### PR DESCRIPTION
**Pull request by Janhouse**
Clicking on external links or any link with middle mouse button opens it in the default browser.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,73 @@
...
+           iframe.contentWindow.document.body &&
+           iframe.contentWindow.document.body.innerHTML) 
+           {
+            //now deal with links
+            iframe.contentWindow.document.body.addEventListener('click', handleLinks, false);
+           } 
+           else 
+           {
+             // not yet, let's wait a bit and try again
+             setTimeout(isLoaded, 300);
```
Comment (passcod) [inline index.html:65]: Is that really the best we can do? Isn't there a way with some event of some kind? (Sorry, don't have time to investigate.)

**Comment by passcod** [comment]
That looks great, although the code's style is not quite what I use (see e.g. [this document](https://github.com/passcod/node-style-guide), main lines are: consistent indentation, using 2 spaces for indenting, braces on the same line like `if (foo) {` and `} else {`, and whitespace consistency like spaces around `=` and within parens etc). But if you don't want to worry with that I can fix it myself, no problem, thanks lots for this PR! You can add yourself to the contributors in the package.json :)

File: index.html [hunk 2]
```diff
@@ -1,11 +1,73 @@
...
-    <title>TweetDeck</title>
-  </head>
-  <body>
-    Connecting...
-    <script>location.replace('https://web.tweetdeck.com')</script>
-  </body>
+ <head>
+	<style>
+	html, body {margin:0;padding:0;}
+	html, body { min-height:100%; }
```
Comment (passcod) [inline index.html:6]: These two lines can be merged into one block.

**Comment by Janhouse** [comment]
Well, it was more of a 5 minute dirty hack to make it work as I wanted to. I thought that proof of concept was better than just a feature request with no code. :)

**Pull request merged**